Dockerfile

```diff
@@ -40,4 +40,3 @@ ENV REPO_TOOL=/aosp/bin/repo
 FROM builder AS builder-dev
 
 ENTRYPOINT ["sh", "/aosp/builder/dev-local/setuplocaluserinvoker.sh"]
-
```

Comment:
These changes in line breaks are useless. Please revert. Here and in the other places

Reply:
Fixed smaller ones. My vim deletes trailing whitespaces!